zeebe/exporters/rdbms-exporter/src/main/java/io/camunda/exporter/rdbms/RdbmsExporter.java

```diff
@@ -205,7 +205,7 @@ private void initializeRdbmsPosition() {
 
   private void flushAndReschedule() {
     flushExecutionQueue();
-    controller.scheduleCancellableTask(Duration.ofSeconds(5), this::flushAndReschedule);
+    controller.scheduleCancellableTask(Duration.ofMillis(100), this::flushAndReschedule);
```

Comment:
:question: Did you lower it for tests, e.g. so it's faster? The simpler way would be to make it configurable. Is flushing every 100ms a good idea in prod? :thinking: Admittedly I might be lacking some context here.